torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -392,6 +392,12 @@ Tensor TensorExprKernel::ComputeValue(torch::jit::Value* v) {
           "aten_round", v, [](const Expr& a) { return round(a); });
     } break;
 
+    case aten::relu: {
```

Comment:
@lly-zero-one I checked in a version of ReLu yesterday. It should be in the repo now:

    case aten::relu: {
      return ComputeOneOperand("aten_relu", v, [](const Expr& a) {
        Expr zero_cond = CompareSelect::make(a, Expr(0.0f), kLT);
        return ifThenElse(zero_cond, Expr(0.0f), a);
      });
    } break;
Though not sure which version is better perf-wise.

Reply:
Thanks, let me pull it out.